addons/ai/functions/fnc_mortarFire.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+    Author:
+        Adam Bridgford
+
+    Description:
+        Fires the mortar based on POS
+
+    Parameter(s):
+        0: Unit <OBJECT>
+
+    Return Value:
+        Nothing
+*/
+
+hint "mortarFired called";
+params ["_unit"];
+
+private _unit = (_this select 0);
+                enPos = _unit findNearestEnemy _unit;
```

Comment:
Not essential, but keeping indentation consistent is good for readability.
Currently this suggests everything below line 18 is in some new scope, which it isn't.